x/protocolpool/keeper/keeper.go

```diff
@@ -397,19 +393,23 @@ func (k Keeper) calculateClaimableFunds(ctx context.Context, recipient sdk.AccAd
 	periodsPassed := int64(timeElapsed) / int64(*budget.Period)
```

Comment:
it looks like we should return here, otherwise we'd be distributing tokens even if 0 periods have passed

Reply:
I don't think this case would be triggered as the above logic ensures that enough time has passed for a period to be considered.